gabbi/tests/test_utils.py

```diff
@@ -80,10 +83,52 @@ def test_extract_content_type_bad_params(self):
         self.assertEqual('text/colorful', content_type)
         self.assertEqual('utf-8', charset)
 
-    def test__colorize_missing_color(self):
+
+class ColorizeTest(unittest.TestCase):
+
+    def test_colorize_missing_color(self):
```

Comment:
On Fri, 9 Oct 2015, FND wrote:

> Oh, why's that?

The method being tested was _colorize.

-- 
Chris Dent                                   http://burningchrome.com/
                                 [...]